packages/static/kairos-overlay-files/files/etc/cos/bootargs.cfg

```diff
@@ -1,7 +1,8 @@
 function setSelinux {
     source (loop0)/etc/os-release
     set baseSelinuxCmd=""
-    if regexp "rockylinux|fedora|almalinux|redhat|tumbleweed" $KAIROS_ARTIFACT; then
+    # opensuse does not bundle regexp module so if we are on leap or tumbleweed we need to do this manually
```

Comment:
this comment will be confusing after we merge this PR. Anyone viewing this file won't understand why we explain the non-usage of `regexp`.